modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/DefinitionNode/DefinitionNode.tsx

```diff
@@ -122,9 +122,9 @@ export function DefinitionNode({
 						baseResourceURL,
 						id!,
 						name,
+						objectDefinitionPermissionsURL,
```

Comment:
@igor-franca Please, declare the parameters of this function as named parameters, so we can follow the alphabetical order here.